libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Analytics/SFSDKSalesforceAnalyticsManager.m

```diff
@@ -37,47 +37,29 @@
 #import "UIDevice+SFHardware.h"
 #import "SFIdentityData.h"
 #import "SFApplicationHelper.h"
-#import "SFAuthenticationManager.h"
 #import <SalesforceAnalytics/SFSDKAILTNTransform.h>
 #import <SalesforceAnalytics/SFSDKDeviceAppAttributes.h>
 #import <SalesforceAnalytics/NSUserDefaults+SFAdditions.h>
 #import "SFSDKAppFeatureMarkers.h"
 #import "SFSDKAppConfig.h"
 
+static NSString * const kAnalyticsUnauthenticatedManagerKey = @"-unauthenticated-";
 static NSString * const kEventStoresDirectory = @"event_stores";
 static NSString * const kEventStoreEncryptionKeyLabel = @"com.salesforce.eventStore.encryptionKey";
 static NSString * const kAnalyticsOnOffKey = @"ailtn_enabled";
 static NSString * const kSFAppFeatureAiltnEnabled = @"AI";
 
 static NSMutableDictionary *analyticsManagerList = nil;
 
-@interface SFSDKAnalyticsTransformPublisherPair : NSObject
-
-@property (nonnull, nonatomic, readonly, strong) id<SFSDKTransform> transform;
-@property (nonnull, nonatomic, readonly, strong) id<SFSDKAnalyticsPublisher> publisher;
-
-- (instancetype)initWithTransform:(id<SFSDKTransform>)transform publisher:(id<SFSDKAnalyticsPublisher>)publisher;
-
-@end
-SFSDK_USE_DEPRECATED_BEGIN
-
-@interface SFSDKSalesforceAnalyticsManager () <SFAuthenticationManagerDelegate>
-
-SFSDK_USE_DEPRECATED_END
-@property (nonatomic, readwrite, strong) SFSDKAnalyticsManager *analyticsManager;
-@property (nonatomic, readwrite, strong) SFSDKEventStoreManager *eventStoreManager;
-@property (nonatomic, readwrite, strong) SFUserAccount *userAccount;
-@property (nonatomic, readwrite, strong) NSMutableArray<SFSDKAnalyticsTransformPublisherPair *> *remotes;
-
-@end
-
 @implementation SFSDKSalesforceAnalyticsManager
 
-+ (instancetype) sharedInstanceWithUser:(SFUserAccount *) userAccount {
-    static dispatch_once_t pred;
-    dispatch_once(&pred, ^{
++ (void)initialize {
+    if (self == [SFSDKSalesforceAnalyticsManager class] && analyticsManagerList == nil) {
         analyticsManagerList = [[NSMutableDictionary alloc] init];
```

Comment:
Ended up moving this into `initialize`, because there were two singleton methods that each needed access to it.